controller/events-controller.js

```diff
@@ -0,0 +1,94 @@
+const Event = require("../models/events");
+const { validationResult } = require("express-validator");
+const dotenv = require("dotenv");
+const mongoose = require("mongoose");
+/*let events = [{
+    "NameofEvent":"Onboarding of the new executives",
+    "Members": "seminire@gmail.com, okeyihedi@yahoo.com, okpara@gmail.com",
+    "Venue":"Upper staff room",
+    "Date": "23/2/2023",
+    "Time":"16:00:00 WAT",
+    "Set reminder": "true"
+}]*/
+//I will need the admin to send reminders to their email
+const {
+    transporter,
+    verifyEmailTemplate,
+    forgotEmailTemplate,
+    passwordSetTemplate,
+    verifiedTemplate,
+  } = require("../util/email");
+
+const getAllEvents = async (req, res) =>{
+        //console.log(events);
+        const events = await Event.find({});
+        res.status(200).json(events);
+}
+
+const getEventById = async (req, res) => {
+    let event;
+    try {
+      const FoundEvents = await Event.findById((event) => event.id ===id)
```

Comment:
i think this is wrong.. did you test the code?

Reply:
yes i did. The code is running